tc/sgx/trusted_worker_manager/enclave/wpe/signup_enclave_wpe.cpp

```diff
@@ -231,10 +231,9 @@ void CreateSignupReportDataWPE(const uint8_t* ext_data,
     ComputeSHA256Hash((const char*) ext_data, ext_data_hash);
 
     // Concatenate hash of public encryption key and hash of extended data
-    strncpy((char*)report_data->d,
-        (const char*) enc_key_hash, SGX_HASH_SIZE);
-    strncat((char*)report_data->d,
-        (const char*) ext_data_hash, SGX_HASH_SIZE);
+    memcpy(report_data->d, enc_key_hash, SGX_HASH_SIZE);
```

Comment:
Similar changes need to be made in CreateSignupReportDataKME() and  CreateReportDataKME() functions in signup_enclave_kme.cpp as well?

Reply:
Yes. Better to use memcpy in this case.